challenges/objects-arrays.js

```diff
@@ -46,20 +70,31 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
 const universities = [];
+for (let i=0; i < graduates.length; i++) {
+  universities.push(graduates[i].university);
+}
+universities.sort();
 console.log(universities)
 
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
-
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
 
 Log the result of your new array. */
 const contactInfo = [];
+for (let i=0; i < graduates.length, i++) {
+  contactInfo.push(graduates[i].first_name + ' ' + graduates[i].email) 
+}
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for (let i=0; i < graduates.length, i++) {
+  if(graduates[i].university.includes('uni') ) {
```

Comment:
This code won't work because You are checking for `uni` and all names in an Array start with capital letter, hence it will skip `Uni` and return empty array. You want to call `lowerCase` before checking the contents.That piece of code solves the problem:
`if (graduates[i].university.toLowerCase().includes("uni"))`